go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1845,6 +1845,30 @@ func commonFlows(bridge *bridgeConfiguration) ([]string, error) {
 							"actions=ct(commit, zone=%d, exec(set_field:%s->ct_mark)), output:%s",
 							defaultOpenFlowCookie, netConfig.ofPortPatch, bridgeMacAddress, config.Default.ConntrackZone,
 							netConfig.masqCTMark, ofPortPhys))
+
+					// IP traffic from the OVN network to the host network should be handled normally by the bridge instead of
+					// being forwarded directly to the NIC (prio=100 flow above). This allows pods in the default network to reach
+					// localnet pods on the same node.
+					if config.Gateway.Mode == config.GatewayModeShared || config.Gateway.Mode == config.GatewayModeLocal {
```

Comment:
Let's just check if gateway is not disabled here with `config.Gateway.Mode != config.GatewayModeDisabled`
Then we do specific things inside.

But maybe we are planning on having a more gateway modes (BGP?)

Reply:
Not sure, but I prefer to be explicit on what's allowed :)

Reply:
fair enough